csrankings-j.csv

```diff
@@ -2274,6 +2276,7 @@ Jun Ma 0010,Nanjing University,http://ics.nju.edu.cn/people/junma,OkMoxEkAAAAJ
 Jun Miyazaki,Tokyo Institute of Technology,http://www.lsc.cs.titech.ac.jp/en,NOSCHOLARPAGE
 Jun Rekimoto,University of Tokyo,http://www.iii.u-tokyo.ac.jp/faculty/rekimoto_junichi,wKJLtK8AAAAJ
 Jun Sakuma,University of Tsukuba,https://www.mdl.cs.tsukuba.ac.jp,v5emswQAAAAJ
+Jun Shen,University of Wollongong,https://scholars.uow.edu.au/jun-shen,Bf6gvGkAAAAJ

```

Comment:
Correct entry should be Jun Shen 0001 (see https://dblp.org/pid/48/5700-1.html).

Reply:
thanks, I fixed